buildtest/cli/config.py

```diff
@@ -154,6 +159,31 @@ def view_configuration(configuration, theme=None, pager=None):
     console.print(syntax)
 
 
+def list_profiles(configuration, theme=None, print_yaml=None):
+    """Display the list of profile for buildtest configuration file.This implements command ``buildtest config profiles list``
```

Comment:
add a space after the period `file. This`

Reply:
done